src/task_run_diagnostic_reports.py

```diff
@@ -85,20 +88,44 @@ def create_ticket(group, disease, touchstone,
         description = get_version_url(report, version, config) if \
             report_success else \
             "Auto-run failed with error: {}".format(error)
-        issue = yt.create_issue(Project(vimc_project_id),
-                                summary.format(group, disease, touchstone),
-                                description)
-        yt.run_command(
-            Command([issue],
-                    "for {} implementer {} tag {} tag {} tag {}".format(
-                        report.assignee,
-                        report.assignee,
-                        group, disease,
-                        touchstone)))
+        create_tags(yt, group, disease, touchstone, report)
+        query = "tag: {} AND tag: {} AND tag: {} AND tag: {} " \
+                "AND state: Incoming"
+        existing_issues = yt.get_issues(
+            query.format(disease, touchstone, group, report.name),
+            ["id", "summary", "description", "tags(name)"])
+        summary = summary.format(group, disease, touchstone)
+        if len(existing_issues) > 0:
+            existing_issue = existing_issues[0]
+            yt.update_issue(existing_issue, summary, description)
+        else:
+            issue = yt.create_issue(Project(vimc_project_id),
+                                    summary,
+                                    description)
+            comm = "for {} implementer {} tag {} tag {} tag {} tag {}".format(
+                report.assignee,
+                report.assignee,
+                group, disease,
+                touchstone,
+                report.name)
+            yt.run_command(
+                Command([issue], comm))
     except Exception as ex:
         logging.exception(ex)
 
 
+def create_tags(yt, group, disease, touchstone, report):
```

Comment:
using tags to query for existing issues, but querying for a non-existent tag returns a 400, so have to make sure all tags are created up front